src/Test/Utilities/Portable/CompilerTraitAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This appears identical to `src/Test/Utilities/Desktop/CompilerTraitAttribute.cs`. Do we need both?